### ISSUE TITLE
List of papers/projects stemming from AstroHackWeek

### ISSUE THREAD

**Issue by jakevdp** [AstroHackWeek/AstroHackWeek2015#27]
We should find a place to keep track of this... where is best? Git repo? Website?

**Comment by jakevdp** [comment]
Maybe the best location is a page on the 2016 website, where we can begin listing/tracking these. @kbarbary, would that be easy to do?

**Comment by kbarbary** [comment]
Making a page under the blog might make more sense. The blog is "permanent", whereas the website changes every year. Could link directly to that page from the 2016 landing page though.

**Comment by drphilmarshall** [comment]
I agree, the blog would be good for this. In an ideal world, each paper
that arose from astro hack week would get its own blog post!

**Comment by jakevdp** [comment]
I was thinking of a page that can be continually updated – so maybe a github wiki page would be better.

**Comment by kbarbary** [comment]
Let's start with a wiki page for the moment. The content will be the hardest part and a wiki will make it easier for more people to fill it in. Either on the `astrohackweek.github.io` or the `blog` repo.

**Comment by jakevdp** [comment]
OK – started it here: https://github.com/AstroHackWeek/blog/wiki

**Comment by jakevdp** [comment]
@davidwhogg @dhuppenkothen @drphilmarshall can you forward this wiki link to folks?
